torchaudio/metrics.py

```diff
@@ -1,9 +1,17 @@
 from typing import List, Union
 
 
-def levenshtein_distance(r: Union[str, List[str]], h: Union[str, List[str]]):
+def levenshtein_distance(r: Union[str, List[str]], h: Union[str, List[str]]) -> int:
     """
     Calculate the Levenshtein distance between two lists or strings.
+
+    The function computes an edit distance allowing deletion, insertion and substitution.
+    The result is an integer. Users may want to normalize by the length of the reference.
```

Comment:
This should imo also include an explanation of what happens when the user passes a list of strings and how that is different.

Reply:
Agree, and updated. Doc may need to be updated further following discussion from [comment](https://github.com/pytorch/audio/pull/927#discussion_r497641380).